docs/docs/pii-management.mdx

```diff
@@ -0,0 +1,184 @@
+or [huggingface](https://huggingface.co/docs/transformers/training) model and install it in your Rasa Pro environment.
+:::
+
+### How to populate the rule_lists section
+
+The `rule_lists` section contains a list of anonymization rule lists. Each rule list must have a unique `id` and a list
+of `rules`. Each rule must have an `entity` field and a `substitution` field. The `entity` field specifies the Presidio
+entity type to be anonymized and must be in uppercase.
+
+The `substitution` field specifies the anonymization method to be used. Currently, the
+following anonymization methods are supported: `text`, `mask`, and `faker`. If no substitution method is specified,
+the default substitution method is `mask`.
+
+The `value` field is only required for the `text` anonymization method. It specifies the text to be used as the anonymized value.
+If the `value` field is not specified, the original entity value to be anonymized will be replaced with a null value.
+
+The `faker` anonymization method uses the [Faker](https://faker.readthedocs.io/en/master/) library to generate fake data.
+The `faker` substitution method does not support the following Presidio entity types:
+- `CRYPTO`, `NRP`, `MEDICAL_LICENSE`
+- `US_BANK_NUMBER`, `US_DRIVER_LICENSE`
+- `UK_NHS`
+- `IT_FISCAL_CODE`, `IT_DRIVER_LICENSE`, `IT_PASSPORT`, `IT_IDENTITY_CARD`
+- `SG_NRIC_FIN`
+- `AU_ABN`, `AU_ACN`, `AU_TFN`, `AU_MEDICARE`
+
+If any of the above entities is used together with the `faker` substitution method, the anonymization pipeline will default
+to the `mask` substitution method.
+
+## How to update the Kafka event broker configuration
+
+The anonymization pipeline uses Kafka to publish the anonymized events to the Kafka topic that is mapped to the
+anonymization rule list. You can configure the Kafka event broker in the `endpoints.yml` file. The Kafka event broker
+must contain the `anonymization_topics` section, which must have the following structure:
+
+```yaml
+event_broker:
+  type: kafka
+  partition_by_sender: True
+  security_protocol: PLAINTEXT
+  url: localhost
+  anonymization_topics:
+    - name: topic_1
+      anonymization_rules: rules_1
+    - name: topic_2
+      anonymization_rules: rules_2
+  client_id: kafka-python-rasa
+```
+
+The `anonymization_topics` section contains a list of Kafka topics to which the anonymized events will be published.
+Each Kafka topic must have a `name` field and an `anonymization_rules` field. The `name` field specifies the name of the
+Kafka topic. The `anonymization_rules` field specifies the `id` of the anonymization rule list to be used for the Kafka topic.
+
+## How to enable anonymization of PII in logs
+
+You can enable anonymization of PII in logs by filling the `logger` section in the `endpoints.yml` file.
+The `logger` section must have the following structure:
+
+```yaml
+logger:
+  formatter:
+    anonymization_rules: rules_1
```

Comment:
can this be also a list of rules?

Reply:
No, this can only be a string and must match and `id` of the anonymization rule list in the `anonymization` section. So all rules are grouped only in that section.